clients/drcachesim/tools/common/decode_cache.h

```diff
@@ -127,7 +131,7 @@ class decode_info_base_t {
  */
 class instr_decode_info_t : public decode_info_base_t {
 public:
-    ~instr_decode_info_t();
+    virtual ~instr_decode_info_t();
```

Comment:
Better to use `override` to ensure parent is virtual too I would think.

Reply:
Done.

Reply:
Also for decode_cache_t